src/tests/api/BavHappyPath.test.ts

```diff
@@ -0,0 +1,54 @@
+import bavStubPayload from "../data/exampleStubPayload.json";
+import { constants } from "../utils/ApiConstants";
+import {
+    getSessionAndVerifyKey,
```

Comment:
We can get rid of the unused imports we don't need. 

- sessionPost
- stubStartPost